docs/kcl/reduce.md

```diff
@@ -9,21 +9,21 @@ Take a starting value. Then, for each element of an array, calculate the next va
 using the previous value and the element.
 
 ```js
-reduce(array: [u64], start: Sketch, reduce_fn: FunctionParam) -> Sketch
+reduce(array: [KclValue], start: KclValue, reduce_fn: FunctionParam) -> KclValue
 ```
 
 
 ### Arguments
 
 | Name | Type | Description | Required |
 |----------|------|-------------|----------|
-| `array` | `[u64]` |  | Yes |
-| `start` | [`Sketch`](/docs/kcl/types/Sketch) | A sketch is a collection of paths. | Yes |
+| `array` | [`[KclValue]`](/docs/kcl/types/KclValue) |  | Yes |
+| `start` | [`KclValue`](/docs/kcl/types/KclValue) | A memory item. | Yes |
```

Comment:
I didn't realize `A memory item.`, the doc comment of `KclValue`, was making it into the user-facing docs. We should probably improve that.